tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1517,7 +1518,7 @@ public void RunCommand_Example_2()
                 database.DropCollection(collectionName);
             }
             database.CreateCollection(collectionName);
-            
+
```

Comment:
Files should not be changed in only whitespace.

It's OK to fix the whitespace if other changes are also being made.